src/test/java/features/authentication/AndroidTest.feature

```diff
@@ -0,0 +1,21 @@
+@Epic=authenticationTest
+Feature: Feature - Android app Login
+
+#  @Regression
+  Scenario: 1562447 - Android app Login successful
+    When user tap on the Login text button
+    And user select and tap on Germany
+    And user enter the username "limangotesting@gmail.com"
```

Comment:
Please extract these into variables using Scenario Outline and Exmaples table

Reply:
updated